security/container_security/security-container-signature.adoc

```diff
@@ -0,0 +1,14 @@
+[id="security-container-signature"]
+= Container image signatures
+
+Red Hat delivers signatures for the images in the Red Hat Container Registries. Those signatures can be automatically verified when being pulled to OpenShift Container Platform (OCP) 4 clusters using Machine Config.
```

Comment:
s/OpenShift Container Platform (OCP) 4/{product-title} 4

You can include "4" (the version) in the first reference, but you don't have to include it in subsequent references. :) 

For all references to OCP:
s/OCP/{product-title}

For Machine Config, if it is the Machine Config Oprerator:
s/Machine Config/Machine Config Operator (MCO)

And if you'd like (totally optional), you can add an additional resources link to the MCO docs.